lib/repl.js

```diff
@@ -1058,7 +1058,7 @@ REPLServer.prototype.defineCommand = function(keyword, cmd) {
     cmd = {action: cmd};
   } else if (typeof cmd.action !== 'function') {
     throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-      'action', 'function', cmd.action);
+                               'action', 'function', cmd.action);
```

Comment:
do we not break on all arguments if we break on any? For some reason I thought that was the more used style throughout the codebase